python/paddle/fluid/layers/metric.py

```diff
@@ -20,6 +20,7 @@
 from ..initializer import Normal, Constant
 from ..framework import Variable
 from ..param_attr import ParamAttr
+import nn
```

Comment:
Maybe we can delete this file and put `accuracy` and `auc` to `evaluator`.

Currently seems "evaluator" is the python wrapping using operators, and `framework/metrics.py` is a pure python implementation.